### ISSUE TITLE
Error with hue keypad

### ISSUE THREAD

**Issue by freaktechnik** [WebThingsIO/homekit-adapter#50]
Now that hue hub pairing works (well, it claims the pin is wrong, but it actually works...) I'm running into the next issue: the keypads seem to have a different value enum than what the adapter expects:

```
2020-11-04 00:52:06.152 INFO   : homekit-adapter: [Device 2E:5A:D9:1D:11:32] Found characteristic: public.hap.characteristic.input-event
2020-11-04 00:52:06.152 INFO   : homekit-adapter: {
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "type": "73",
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "iid": 588410585204,
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "value": null,
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "format": "uint8",
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "minValue": 0,
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "maxValue": 0,
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "minStep": 1,
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   "perms": [
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     "pr",
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     "ev"
2020-11-04 00:52:06.152 INFO   : homekit-adapter:   ]
2020-11-04 00:52:06.152 INFO   : homekit-adapter: }
2020-11-04 00:52:06.152 INFO   : homekit-adapter: Error loading device 2E:5A:D9:1D:11:32: Error: Invalid enum value
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     at HomeKitProperty.fromHap (/home/node/.mozilla-iot/addons/homekit-adapter/lib/property-utils.js:15:11)
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     at HomeKitProperty.setCachedValue (/home/node/.mozilla-iot/addons/homekit-adapter/lib/homekit-property.js:51:20)
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     at new HomeKitProperty (/home/node/.mozilla-iot/addons/homekit-adapter/lib/homekit-property.js:41:10)
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     at HomeKitDevice.addProperty (/home/node/.mozilla-iot/addons/homekit-adapter/lib/homekit-device.js:1486:20)
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     at HomeKitDevice.parseAccessories (/home/node/.mozilla-iot/addons/homekit-adapter/lib/homekit-device.js:367:33)
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     at /home/node/.mozilla-iot/addons/homekit-adapter/lib/homekit-device.js:282:21
2020-11-04 00:52:06.152 INFO   : homekit-adapter:     at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

Which presumably means the enum in https://github.com/WebThingsIO/homekit-adapter/blob/master/lib/homekit-device.js#L1479-L1502 has a value that the keypad doesn't support?

**Comment by mrstegeman** [comment]
Seems like the device isn't being reported properly, given these bits:

```
   "value": null,
   "minValue": 0,
   "maxValue": 0,
```

**Comment by mrstegeman** [comment]
Should be fixed in v0.10.1.

**Comment by freaktechnik** [comment]
Confirmed fixed. Now interestingly the only properties that have accurate values are the battery levels, but that's an issue for another time.
